src/diffusers/loaders.py

```diff
@@ -1109,8 +1187,24 @@ def _remove_text_encoder_monkey_patch_classmethod(cls, text_encoder):
                 attn_module.v_proj = attn_module.v_proj.regular_linear_layer
                 attn_module.out_proj = attn_module.out_proj.regular_linear_layer
 
+        if getattr(text_encoder, "aux_state_dict_populated", False):
+            for _, aux_module in text_encoder_aux_modules(text_encoder):
+                if isinstance(aux_module.fc1, PatchedLoraProjection):
+                    aux_module.fc1 = aux_module.fc1.regular_linear_layer
+                    aux_module.fc2 = aux_module.fc2.regular_linear_layer
+
+            text_encoder.aux_state_dict_populated = False
```

Comment:
If this is the only other part of the code that requires knowing if the mlp blocks were patched, I don't think we need the flag to know, we can just check the class of the mlp blocks, no?